src/data/fintype/basic.lean

```diff
@@ -1,7 +1,7 @@
 /-
 Copyright (c) 2017 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Author: Mario Carneiro
+Author: Mario Carneiro, Kenny Lau, Yakov Pechersky
```

Comment:
@pechersky 
We don't have any explicit guidelines about this yet. It's on our to-do list. But the `Author` line in files is generally not a list of everyone who edits a file. (This is tracked by git and easily seen on GitHub, e.g. "13 contributors" at the top of https://github.com/leanprover-community/mathlib/blob/master/src/data/fintype/basic.lean .)

I tend to think of the authorship attribution as an answer to "who should I ask if I have questions about the development in this file?" Adding a large section or doing a significant refactor are reasons to update the list in this interpretation. Adding one line to a 1000 line file isn't.

I'm certainly not trying to steal credit from you (again, it's tracked by git) and I'm not saying you should have known this! But if it's alright with you, I think it would be better to remove the authorship updates in this PR.

Reply:
That totally makes sense. It does not seem like any credit is being unassigned. Using the commit info to keep track of authorship was my default approach. However, the docs at https://github.com/leanprover-community/leanprover-community.github.io/blame/451f41a6f5f7c6c10d14d4617a59fda3fd8a576e/templates/contribute/style.md#L32 were not clear about what constituted `worked on the file`. Since that header mentions the Apache license, and I am not familiar with that license's particularities, I erred on the side of including authorship info. It's not a problem to remove this commit, and I'll undo some authorship changes I made in other recent PRs.

Reply:
Thanks for understanding! Those docs were probably written when there were ~5 contributors instead of the ~90 now. Clearly they need to be updated.